presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,59 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Add spatial join (broadcast and distributed) support for :func:`ST_Equals`,
+  :func:`ST_Overlaps`, :func:`ST_Crosses`, and :func:`ST_Touches`.
+* Fix a bug where index joins would fail with the error "driver should never block".
```

Comment:
It's unclear to me what does it mean by
```
Now queries will wait until a configured timeout for the indexes to load.
```
Are the indexes loaded once per query or once per worker?

Reply:
I think it means 
```
Queries will error out if the index is unable to load within the specified timeout.
```
Does the above line look more understandable?

Also I think the index is loaded by each driver thread (if that's the worker you're talking about).

cc @rschlussel

Reply:
We can remove `only` if that's confusing.